### DISCUSSION TITLE
Entrance blocked off.

### DISCUSSION THREAD

**Discussion by ThyGoodBoy** [FakeFishGames/Barotrauma#13757] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Separatists' biome entrance towards 2nd biome not connected to the 1st biome.

### Reproduction steps

1. Started a singleplayer campaign. Unmodded, 
[BestBoy.zip](https://github.com/FakeFishGames/Barotrauma/files/14920540/BestBoy.zip)
2. Reached the location from where the separatists entrance towards 2nd biome is visible, and where can be seen entrance being unconnected to the first biome.
![image](https://github.com/FakeFishGames/Barotrauma/assets/158489743/a5141b4a-5fa5-45f1-b3f1-36f27d7d1502)

### Bug prevalence

Just once

### Single player or multiplayer?

Single player

### Version

v1.3.0.2

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Closing as a duplicate of https://github.com/FakeFishGames/Barotrauma/discussions/13791
